content/consul/v1.21.x/content/docs/manage/scale/autopilot.mdx

```diff
@@ -1,51 +1,26 @@
 ---
 layout: docs
-page_title: Consul autopilot
+page_title: Consul Autopilot
 description: >-
   Use Autopilot features to monitor the Raft cluster, introduce stable servers, and clean up dead servers.
 ---
 
-# Consul autopilot
+# Consul Autopilot
 
-This page describes Consul autopilot, which supports automatic, operator-friendly management of Consul
-servers. It includes cleanup of dead servers, monitoring the state of the Raft
-cluster, and stable server introduction.
+This page describes Consul Autopilot, which supports automatic, operator-friendly management of Consul servers. Autopilot helps maintain the health and stability of the Consul server cluster by monitoring server health, introducing stable servers, and cleaning up dead servers. Furthermore, Consul Enterprise customers can leverage two additional Autopilot features, namely redundancy zones and automated upgrades to enhance datacenter resiliency and simplify operations.
 
-To use autopilot features (with the exception of dead server cleanup), the
-[`raft_protocol`](/consul/docs/reference/agent/configuration-file/raft#raft_protocol)
-setting in the Consul agent configuration must be set to 3 or higher on all
-servers. In Consul `0.8` this setting defaults to 2; in Consul `1.0` it will
-default to 3. For more information, check the [Version Upgrade
-section](/consul/docs/upgrade/version-specific) on Raft protocol
-versions in Consul `1.0`.
+## Overview
 
-In this tutorial you will learn how Consul tracks the stability of servers, how
-to tune those conditions, and get some details on the other autopilot's features.
+Autopilot includes the following features:
```

Comment:
```suggestion
Consul autopilot helps you maintain the health and stability of the Consul server cluster. It includes the following features:
```
Moved the sentence so that there's less at the top and more before the list.